src/plugins/collapsible-alerts/base.scss

```diff
@@ -34,6 +34,10 @@ details {
 					text-decoration: underline;
 				}
 			}
+
+			h2{
+				font-size: x-large;
+			}
```

Comment:
This change might be better off in a separate PR since it's unrelated to link revisions.